client/dashboard/domains/dataviews/field-domain-name.tsx

```diff
@@ -36,17 +36,21 @@ export const DomainNameField = ( {
 			<VStack spacing={ 1 }>
 				<span style={ textOverflowStyles }>{ value }</span>
 				{ showPrimaryDomainBadge && domain.primary_domain && (
-					<span
-						style={ {
-							...textOverflowStyles,
-							color: 'var(--dashboard__foreground-color-success)',
-							fontWeight: 'normal',
-							textDecoration: 'underline',
-							textDecorationStyle: 'dotted',
-						} }
+					<Tooltip
+						text={ __( 'The site address your visitors will see while viewing your site.' ) }
```

Comment:
```suggestion
						text={ __( 'The site address your visitors see while viewing your site.' ) }
```

Can we more "will" from this sentence?

Reply:
Maybe even shorter?

```suggestion
						text={ __( 'The address visitors see on your site.' ) }
```

Reply:
Yeah - it's a mouthful. I got the copy from https://github.com/Automattic/wp-calypso/pull/106849#issuecomment-3478783894 but I agree it can be simplified.

What would you say about:
> The address people will see when visiting your site.

Reply:
It's better! I still think it's a longer than it needs to be. I ultimately think the problem is we're using the word "primary site address" when "public url" should probably do the trick (maybe naive), but I see in the originating issue we've agreed to scope this down to your PR here. 

As long as we keep it as short as possible. Hover tooltips shouldn't be responsible for communicating important, primary-type information, but if we have to use it, always go as short and concise as possible. If we can't communicate what we want in a few words, something is definitely off with our UX.

Reply:
Okay, I'll merge this as it is then and we can discuss and revisit changing anything else if necessary.

The terminology (domain, subdomain, address, URL, etc.) is confusing for a lot of people including us so changing it should be really thought through. For example for a long time we used "Primary domain" and then changed it to "Primary site address" because it does not create ambiguity with subdomains which can also be set as primary. The current term follows our domains naming guidelines - p58i-nfJ-p2 .